.github/workflows/ci-code.yml

```diff
@@ -127,13 +126,18 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
```

Comment:
this is a short run that you're multiplying here, right?

perhaps reduce the max runtime then

Reply:
sure ting